### PR TITLE
Update installer path - part 2 (improve hariktgh-4)

### PR DESCRIPTION
**Pull request by Radon8472**
This PR changes the check for a successful install like diskussed in [harikt#4 (comment)].

Additional it fixes as minor issue with dev-depencies to calm dependabot (details in 277d099 and b27babc).

I recommend creating a versions-tag for 1.1.1 after merging this.

[harikt#4 (comment)]: https://github.com/harikt/pwmoduleinstaller/pull/4#issuecomment-1106044331

### CODE REVIEW

**Comment by Radon8472** [comment]
@harikt can you accept this, or tell me what I should change that you can accept it?

File: composer.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
         "composer-plugin-api": "^1.0 || ^2.0"
     },
     "require-dev": {
-        "composer/composer": "1.0.* || ^2.0"
+        "composer/composer": "^1.10.23 || ^2.1.9"
```
Comment (harikt) [inline composer.json:34]: We don't need to worry about the dependent bots. I guess we are good with `^1.0 || ^2.0` .

**Review by harikt — COMMENTED** [review]
I am sorry for the delay. I didn't noticed the PR itself.

I guess you can change the composer requirement to the comments as mentioned and we can merge and do a release.

File: composer.json (hunk 1 above)
Comment (harikt) [reply composer.json:34]: Also this is a dev dependency not a requirement itself. So nothing to worry about.

Comment (Radon8472) [reply composer.json:34]: I know that dev-dependencies basicly never impacs the ussage with this project. But github depedabot was complaining, so I fixed it, that this repo is clean and git does not send warnings.

**Comment by Radon8472** [comment]
> I am sorry for the delay. I didn't noticed the PR itself.
> 
> I guess you can change the composer requirement to the comments as mentioned and we can merge and do a release.

What comments should I change?

I am not realy sure what you mean.

**Pull request merged**